modules/gis/src/lib/geojson-to-binary.ts

```diff
@@ -1,10 +1,17 @@
-// Convert GeoJSON features to flat binary arrays
-// @ts-nocheck
+import {Feature, GeoJsonProperties} from 'geojson';
+import {BinaryFeatures, BinaryPointFeatures, BinaryLineFeatures, BinaryPolygonFeatures} from '../types';
 
-export function geojsonToBinary(features, options = {}) {
+export type GeojsonToBinaryOptions = {
+  coordLength?: number;
+  numericPropKeys?: string[];
+  PositionDataType?: Function;
+}
+
+/** Convert GeoJSON features to flat binary arrays */
+export function geojsonToBinary(features: Feature[], options: GeojsonToBinaryOptions = {}): BinaryFeatures {
   const firstPassData = firstPass(features);
   return secondPass(features, firstPassData, {
-    coordLength: options.coordLength || firstPassData.coordLength || 2,
```

Comment:
Note the `|| 2` is from this discussion https://github.com/visgl/loaders.gl/pull/1235. It might still be necessary?